Phishing Campaign - Add/Remove incident (#31678)
demisto/content: Packs/Campaign/ReleaseNotes/3_4_0.md

Reviewer: mention replacing the previous options with the new ones

Author: What do you think.....?
- **Braking Change:** Replaced the "Link" Button to "Add to campaign" Button to manually add incident(s) to a campaign using the incident id.
- **Braking Change:** Replaced the "Unlink" Button to "Remove from Campaign" Button to remove manually incident(s) from a campaign using the incident id.
- **Braking Change:** Replaced the "Link & Close" Button to "Add To Campaign & Close". This will add the incident to the campaign and close the incident.

#### Scripts

##### GetCampaignIncidentsInfo

- Documentation and metadata improvements.

##### ShowCampaignIncidentsOwners

- Fix an issue where `Incident Owners` section shown an error when is no incidents in the context data.

##### ShowCampaignLastIncidentOccurred

- Fix an issue where `Last Incident Occurred` section shown an error when is no incidents in the context data.

##### ShowCampaignRecipients

- Fix an issue where `Campaign Recipients` section shown an error when is no incidents in the context data.

##### ShowCampaignSenders

- Fix an issue where `Campaign Senders` section shown an error when is no incidents in the context data.

##### ShowCampaignSimilarityRange

- Fix an issue where `Similarity Range` section shown an error when is no incidents in the context data.

##### GetCampaignIndicatorsByIncidentId
- Updated the Docker image to: *demisto/pandas:1.0.0.83429*.

- Fix an issue where `Mutual Campaign Indicators` section shown an error when is no incidents in the context data.

##### PerformActionOnCampaignIncidents

> - Added the "Add to campaign" Button to add manually incident(s) to a campaign using the incident id.

Reviewer: ```suggestion
- Added the "Add to campaign" Button to manually add incident(s) to a campaign using the incident id.
```

Author: Done

#### Scripts

##### GetCampaignIncidentsInfo

> - Documentation and metadata improvements.
> 
> ##### ShowCampaignIncidentsOwners
> 
> - Fixed an issue where `Incident Owners` section raised an error when no incidents were present in the context data.
> 
> ##### ShowCampaignLastIncidentOccurred
> - Fixed an issue where `Last Incident Occurred` section raised an error when no incidents were present in the context data.
> 
> ##### ShowCampaignRecipients
> 
> - Fixed an issue where `Campaign Recipients` section raised an error when no incidents were present in the context data.

Reviewer 2: ```suggestion
Documentation and metadata improvements.

##### ShowCampaignIncidentsOwners

Fixed an issue where the `Incident Owners` section raised an error when no incidents were present in the context data.

##### ShowCampaignLastIncidentOccurred
Fixed an issue where the `Last Incident Occurred` section raised an error when no incidents were present in the context data.

##### ShowCampaignRecipients

Fixed an issue where the `Campaign Recipients` section raised an error when no incidents were present in the context data.
```

##### ShowCampaignSenders

> - Fixed an issue where `Campaign Senders` section raised an error when no incidents were present in the context data.
> 
> ##### ShowCampaignSimilarityRange
> 
> - Fixed an issue where `Similarity Range` section raised an error when no incidents were present in the context data.

Reviewer 2: ```suggestion
Fixed an issue where the `Campaign Senders` section raised an error when no incidents were present in the context data.

##### ShowCampaignSimilarityRange

Fixed an issue where the `Similarity Range` section raised an error when no incidents were present in the context data.
```

##### GetCampaignIndicatorsByIncidentId
- Updated the Docker image to: *demisto/pandas:1.0.0.83429*.

> - Fixed an issue where `Mutual Campaign Indicators` section raised an error when no incidents were present in the context data.

Reviewer 2: ```suggestion
- Fixed an issue where the `Mutual Campaign Indicators` section raised an error when no incidents were present in the context data.
```

##### PerformActionOnCampaignIncidents

> - **Braking Change:** Replaced the "Link" Button to "Add to campaign" Button to manually add incident(s) to a campaign using the incident id.
> - **Braking Change:** Replaced the "Unlink" Button to "Remove from Campaign" Button to remove manually incident(s) from a campaign using the incident id.
> - **Braking Change:** Replaced the "Link & Close" Button to "Add To Campaign & Close". This will add the incident to the campaign and close the incident.

Reviewer 2: ```suggestion
**Breaking Changes:**
- Replaced the "Link" button with the "Add to campaign" button to manually add incident(s) to a campaign using the incident ID.
- Replaced the "Unlink" button with the "Remove from Campaign" button to manually remove incident(s) from a campaign using the incident ID.
- Replaced the "Link & Close" button with the "Add To Campaign & Close" button. This will add the incident to the campaign and close the incident.
```

#### Incident Fields

- **Actions On Campaign Incidents**

> - **Braking Change:** Replaced the "Link" Button to "Add to campaign" Button.
> - **Braking Change:** Replaced the "Unlink" Button to "Remove from Campaign" Button.
> - **Braking Change:** Replaced the "Link & Close" Button to "Add To Campaign & Close".

Reviewer 2: ```suggestion
- **Breaking Changes**
   - Replaced the "Link" button with the "Add to campaign" button.
   - Replaced the "Unlink" button with the "Remove from Campaign" button.
   - Replaced the "Link & Close" button with the "Add To Campaign & Close" button.
```

#### Playbooks

##### Detect & Manage Phishing Campaigns

> - Added new value to the default value for the `fieldsToDisplay` argument.

Reviewer 2: ```suggestion
Added a new default value for the `fieldsToDisplay` argument.
```
